ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/SignatureIntegrationForm.tsx

```diff
@@ -63,6 +73,31 @@ const VerificationExpandableSection = ({ toggleText, children }): ReactElement =
     );
 };
 
+function regularExpressionIcon(): ReactElement {
+    return (
+        <Popover
+            bodyContent={
+                <div>
+                    <a href="https://golang.org/s/re2syntax" target="_blank" rel="noreferrer">
+                        See RE2 syntax here
+                    </a>
+                </div>
+            }
+            headerContent={'Supports regular expressions'}
+        >
+            <button
+                type="button"
+                aria-label="More info for input"
+                onClick={(e) => e.preventDefault()}
+                aria-describedby="simple-form-name-01"
```

Comment:
This looks like copy paste from here.
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Integrations/IntegrationForm/AnnotationKeyLabelIcon.tsx#L21

* Bravo for following precedents!
* Tomorrow I will investigate `aria-describedby` prop. While finding and fixing accessibility issues, I have seen axe DevTools report references to `id` props that do not exist.

Reply:
Alternative would be I can probably have the field name as parameter and use it here instead, both within the `describedby` or within the label.

Reply:
You can leave it as is for this contribution. It does not cause an accessibility issue.

I will investigate independently now that accessibility has become a business requirement for 4.6